vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextArea.java

```diff
@@ -63,24 +75,12 @@ public class TextArea extends GeneratedVaadinTextArea<TextArea, String>
      * Constructs an empty {@code TextArea}.
      */
     public TextArea() {
-        this(true);
-    }
+        super("value", "", false);
 
-    /**
-     * Constructs an empty {@code TextArea}.
-     * <p>
-     * If {@code isInitialValueOptional} is {@code true} then the initial value
-     * is used only if element has no {@code "value"} property value, otherwise
-     * element {@code "value"} property is ignored and the initial value is set.
-     *
-     * @param isInitialValueOptional
-     *            if {@code isInitialValueOptional} is {@code true} then the
-     *            initial value is used only if element has no {@code "value"}
-     *            property value, otherwise element {@code "value"} property is
-     *            ignored and the initial value is set
-     */
-    private TextArea(boolean isInitialValueOptional) {
-        super("", "", false, isInitialValueOptional);
+        if (getElement().getProperty("value") == null) {
+            setModelValue("", false);
+            setPresentationValue("");
+        }
```

Comment:
Previously, this private `TextArea` constructor was pointing to this one on the `GeneratedVaadinTextArea` class:

https://github.com/vaadin/flow-components/blob/8db01ab64eea9b9614505f94c4502cea79e632ca/vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/GeneratedVaadinTextArea.java#L830-L837

which only has a call to `setPresentationValue`.  

Just to clarify if this difference was intentional?

Reply:
Thanks, removed. This code shouldn't be needed. See e.g. `RichTextEditor`:

https://github.com/vaadin/flow-components/blob/8db01ab64eea9b9614505f94c4502cea79e632ca/vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java#L116-L118